### ISSUE TITLE
Collision on port 9200 

### ISSUE THREAD

**Issue by Tataukee** [tubearchivist/tubearchivist#332]
### Installation instructions

- [X] I have read and understand the [installation instructions](https://github.com/tubearchivist/tubearchivist#installing-and-updating).
- [X] My issue is not described in the [potential pitfalls](https://github.com/tubearchivist/tubearchivist#potential-pitfalls) section.

### Operating System

Synology

### Your installation or setup question

I have a port conflict on ElasticSearch container and would not be able to use 9200 .
I have tried 9201 and of course changed the variable in the TubeArchivist Container Environnement to : "ES_URL=http://tubearchivist-es:9201/" .
The container runs at first but it just keeping Restarting , it didn't like that apparently .
is there any variable to add in the TubeArchiovist-es container to allow it running over the port 9201 ?
Otherwise please any help would be very appreciative .

### Relevant log output

```shell
waiting for elastic search to start
waiting for elastic search to start
waiting for elastic search to start
waiting for elastic search to start
waiting for elastic search to start
waiting for elastic search to start
waiting for elastic search to start
waiting for elastic search to start
waiting for elastic search to start
waiting for elastic search to start
waiting for elastic search to start
waiting for elastic search to start
Q Search
Stream
stdout
stdout
stdout
stdout
stdout
stdout
stdout
stdout
stdout
stdout
stdout
stdout
```

**Comment by bbilly1** [comment]
You have a few options. I'd suggest to use separate networks for your docker projects, then put a reverse proxy in front of that. That's a quite permanent fix also for feature similar problems and you don't depend on each service to give you the flexibility to configure everything manually.

But of course, Elasticsearch gives you all the flexibility and has extensive documentation on how to configure custom networking ports for example: https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-network.html

So basically from your example above, pass `http.port=9201` as an env var to the ES container, make sure the ports there are correct too, and as you have done above, adapt the `ES_URL` for the Tube Archivist container. 

But all in all, that's not really a Tube Archivist related question, but an Elasticsearch question...
